.circleci/config.yml

```diff
@@ -32,15 +32,27 @@ commands:
       - checkout
       - restore_cache:
           keys:
-            - v3-dependencies-{{ checksum "package.json" }}
+            - v3-dependencies-{{ checksum "yarn.lock" }}
```

Comment:
Unrelated bug fix — the `node_modules` cache should have been keyed by `yarn.lock`, not `package.json`.